src/lib/jsonFetcher.js

```diff
@@ -1,4 +1,4 @@
-var jQuery = ($) ? $ : require('jquery') // funny little hack, sometimes it's in the global scope, sometimes it's maybe not...
+var jQuery = (typeof $ !== 'undefined') ? $ : require('jquery') // funny little hack, sometimes it's in the global scope, sometimes it's maybe not...
 import extra_storage from './extra_storage'
```

Comment:
@BJK Should we eventually transition this to CamelCase to match the preferred React style?